StylableSwiftUI/Classes/Stylist.swift

```diff
@@ -56,12 +59,8 @@ public class Stylist: ObservableObject {
             }
         }
 
-        // Append and sort new styles
-        styles.append(contentsOf: newStyles)
-        styles.sort { $0.identifier < $1.identifier }
-
         // Publish our changes
-        self.styles = styles
+        self.styles = self.matchingMode.insert(styles: newStyles, into: styles)
```

Comment:
Should the insert here also deal with duplicates (i.e. lines 56-60 remove a style if it exists already)?